.github/actions/goreleaser-build-sign-publish/README.md

```diff
@@ -0,0 +1,125 @@
+# goreleaser-build-sign-publish
+> goreleaser wrapper action
+
+## workflows
+
+### build publish
+
+```yaml
+name: goreleaser
+
+on:
+  push:
+    tags:
+      - "v*"
+
+jobs:
+  goreleaser:
+    runs-on: ubuntu-latest
```

Comment:
```suggestion
    runs-on: ubuntu-latest
    env:
    	MACOS_SDK_VERSION: 12.3
```

We should make it easy to change the SDK version we're going to use within this job